docs/tutorials/reuse-ssh-connections.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Reuse SSH Connections
```

Comment:
What's the verdict on this?  If you want a different title and/or filename, let me know what they should be changed to.